src/entities/Project.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Entity,
+  JoinColumn,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToOne,
+  CreateDateColumn,
+  UpdateDateColumn
+} from "typeorm";
+import { Length, IsNotEmpty } from "class-validator";
+
+import { User } from "./User";
+
+@Entity("projects")
+export class Project {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  @Length(4, 100)
+  name: string;
+
+  @ManyToOne(_type => User)
+  @JoinColumn({ name: "user_id" })
+  user!: User;
```

Comment:
user is a very common name. We should give a specific name such as responsibleUserId, createdBy, etc.